server/sandbox/sandbox.go

```diff
@@ -1,4 +1,4 @@
-package server
+package sandbox
```

Comment:
@14rcole We're going to be leaving Sandboxes in the API, though we eventually want to refactor them to be less Kubernetes specific. `libkpod` is fine place, though it should live in a subpackage.